webapp/components/interop.html

```diff
@@ -255,28 +255,24 @@
       this.onSearchCommit = this.handleSearchCommit.bind(this);
     }
 
-    connectedCallback() {
-      super.connectedCallback();
-      this.shadowRoot.querySelector('test-search')
-        .addEventListener('commit', this.onSearchCommit);
-    }
-
-    disconnectedCallback() {
+    detached() {
```

Comment:
Note that `this.shadowRoot` is only available once `super.ready()` has been called (and child elements have been created). The tests were failing preceding this change.